package.json

```diff
@@ -4,13 +4,18 @@
   "description": "Standalone WWW client for Scratch",
   "scripts": {
     "start": "node ./dev-server/index.js",
-    "test": "npm run test:lint && npm run build && npm run test:tap",
+    "test": "npm run test:lint && npm run build && npm run test:unit:tap",
     "test:lint": "eslint . --ext .js,.jsx,.json",
-    "test:smoke": "tap ./test/integration/smoke-testing/*.js --timeout=3600 --no-coverage -R classic",
-    "test:smoke:verbose": "tap ./test/integration/smoke-testing/*.js --timeout=3600 --no-coverage -R spec",
-    "test:smoke:sauce": "SMOKE_REMOTE=true tap ./test/integration/smoke-testing/*.js --timeout=60000 --no-coverage -R classic",
-    "test:tap": "tap ./test/{unit,localization}/*.js --no-coverage -R classic",
-    "test:coverage": "tap ./test/{unit,localization}/*.js --coverage --coverage-report=lcov",
+    "test:integration": "npm run test:integration:jest && npm run test:smoke",
+    "test:integration:jest": "jest ./test/integration/*.test.js",
+    "test:integration:remote": "npm run test:integration:jest && npm run test:smoke:sauce",
+    "test:smoke": "tap ./test/integration-legacy/smoke-testing/*.js --timeout=3600 --no-coverage -R classic",
+    "test:smoke:verbose": "tap ./test/integration-legacy/smoke-testing/*.js --timeout=3600 --no-coverage -R spec",
+    "test:smoke:sauce": "SMOKE_REMOTE=true tap ./test/integration-legacy/smoke-testing/*.js --timeout=60000 --no-coverage -R classic",
+    "test:unit": "npm run test:unit:jest && npm run test:unit:tap",
+    "test:unit:jest": "jest ./test/unit/*.test.js && jest ./test/localization/*.test.js",
+    "test:unit:tap": "tap ./test/{unit-legacy,localization-legacy}/*.js --no-coverage -R classic",
+    "test:coverage": "tap ./test/{unit-legacy,localization-legacy}/*.js --coverage --coverage-report=lcov",
```

Comment:
do we also want to run a jest coverage test, or is there no such thing?

Reply:
I think this should be added in a follow up PR.  It's turning into a bigger project getting it set up than I originally thought and I'd like to get this in so we can start writing tests.  I filed an issue for this work: https://chromebookdb.com/chromebook/toshiba-chromebook-2-cb35-c3350/